### ISSUE TITLE
Request ubuntu-18.04-arm64 for parallels

### ISSUE THREAD

**Issue by jhgorse** [chef/bento#1401]
### Describe the Enhancement:
An ubuntu-18.04-arm64 recipe for parallels.

### Describe the Need:
Apple M1 Parallels users supporting legacy Ubuntu 18.04 x86 based applications and toolsets.

### Current Alternative
Custom hand-built Parallels VM of Ubuntu 18.04 via network install. NOTE: Network or USB media install is important here since Parallels does not support the default installation media used by the Ubuntu 18.04 iso image which uses SATA drivers: https://forum.parallels.com/threads/not-possible-to-install-ubuntu-18-04-arm-64.356282/

### Can We Help You Implement This?:
I would love to do this and make a pull request. However, I am blocked by packer. It does not detect my ParallelsVirtualizationSDK installation.

https://github.com/hashicorp/packer-plugin-parallels/issues/36

**Comment by jhgorse** [comment]
Packer issue is resolved. Packer changed input formats and no longer recognizes the json in the templates directory and they now require the explicit inclusion of the parallels plugin.

xref: https://github.com/chef/bento/issues/1364

There are two paths. Downgrade `packer`, do the mod, send the PR. Or upgrade the `packer` scripts, do the mod, send the PR.

It doesn't really make sense for me to fix this in the old `packer`. I will fork and have my changes for ubuntu-20 and ubuntu-18. We are effectively blocked by #1364.

**Comment by Stromweld** [comment]
Current release of bento packer code has been re-written for hcl2. aarch64 pkrvars files have been added for most OS's. aarch64 builds are still in development and we welcome any testing/PR's for fixes to the builds.

**Comment by Stromweld** [comment]
once builds are working for aarch64 machines we will add them to vagrant cloud.

**Comment by Stromweld** [comment]
I currently have build agents for all providers and both intel and aarch64 now. Unfortunately the current template doesn't appear to be building correctly. I have many to go though to fix, but any help you can offer would be greatly appreciated.

**Comment by fatbasstard** [comment]
Hi! Is there any news about these? Looking forward to this support. Moving to Ubuntu 22 soon but currently still need to maintain Ubuntu 18 installations. Now that I moved to a M1 device, this would be really nice to have!

**Comment by Stromweld** [comment]
18.04 arm builds aren't working but 22.04 and newer are at the moment. it is on my radar, but may be awhile before I can get to this. I'm more working on this project as a side gig when I'm not as busy with my primary role.

**Comment by Stromweld** [comment]
Since this OS is EOL and being removed in the next version of Bento I'm going to close this request. Please see previous version template files if you'd like to build your own box.
